depends/common/ogg/CMakeLists.txt

```diff
@@ -3,20 +3,20 @@ project(ogg)
 cmake_minimum_required(VERSION 2.8)
 include(CheckCCompilerFlag)
 
-CHECK_C_COMPILER_FLAG("-fPIC" HAVE_FPIC)
-if(HAVE_FPIC)
+if(NOT WIN32)
```

Comment:
makes sense. Currently testing it rebased on your branch. http://jenkins.kodi.tv/job/BINARY-ADDONS/77/console